Nos/Views/Components/Button/ActionButton.swift

```diff
@@ -125,8 +128,7 @@ struct ActionButtonStyle: ButtonStyle {
             configuration.label
                 .foregroundColor(.white)
                 .font(.body)
-                .padding(.vertical, 8)
```

Comment:
These hard-coded padding values made it impossible to increase the height of the Delete Account button, so I made them an overridable default value instead.